google_photos_takeout_helper/__main__.py

```diff
@@ -574,6 +597,24 @@ def copy_to_target_and_divide(file: Path):
             file_function=copy_to_target,
             filter_fun=lambda f: (is_photo(f) or is_video(f))
         )
+    if not args.keep_duplicates:
+        print('=====================')
+        print('Removing duplicates...')
+        print('=====================')
+        remove_duplicates(
+            dir=FIXED_DIR
+        )
+
+    print('=====================')
+    print('Populate albums...')
+    print('=====================')
+    for_all_files_recursive(
+        dir=PHOTOS_DIR,
+        folder_function=populate_album_map
+    )
+
+    with open(str(FIXED_DIR) + '/albums.json', 'w+') as outfile:
```

Comment:
Can I move this to input folder, just like stats files? I want to keep output folder *super clean*

Btw, pls stop force pushing, I won't be able to commit anything

Reply:
Sure! Don't forget to update the reader. Also sorry lol I wasn't aware you would be working off this fork 😅.

On my other projects we aim to encapsulate a unit of logical change to a PR and therefore try to make only one or two commit that ultimately gets merged into master but I know that's not the flow here anyways. I'll stop that then.

Reply:
Woah woah, I've just read new edits in readme - this was section *for dummies* - let's keep it as straightforward as possible

I will add some mentions about new options, but in other place

Reply:
Sounds good. You want to make that change then?

Reply:
Already did :+1: